console/console-init/ui/src/modules/connection/containers/ConnectionContainer/ConnectionContainer.tsx

```diff
@@ -84,22 +111,66 @@ export const ConnectionContainer: React.FunctionComponent<IConnectionProps> = ({
         receivers: getFilteredValue(connection.metrics, "enmasse_receivers"),
         status: "running",
         name: connection.metadata.name,
-        creationTimestamp: connection.metadata.creationTimestamp
+        creationTimestamp: connection.metadata.creationTimestamp,
+        selected:
+          selectedConnections.filter(({ name, containerId }) =>
+            compareObject(
+              { name, containerId },
+              {
+                name: connection.metadata.name,
+                containerId: connection.spec.containerId
+              }
+            )
+          ).length === 1
       }));
+    // console.log("list", connectionList[0].selected);
     return connectionList;
   };
 
   const onSort = (_event: any, index: any, direction: any) => {
     setSortBy({ index: index, direction: direction });
     setSortValue({ index: index, direction: direction });
   };
+
+  const onCloseAllConnection = async (connection: IConnection) => {
+    const onCloseConnection = () => {
+      if (connection) {
+        const variables = {
+          cons: [
+            {
+              name: connection.name,
+              namespace: namespace
+            }
+          ]
+        };
+        if (variables) {
```

Comment:
I don't think so we need if condition here, on top we have already check for connection.
Can you please add await as function is async.
await setCloseConnectionQueryVariables(variables);